.drone/config.yaml

```diff
@@ -0,0 +1,48 @@
+---
+kind: pipeline
+type: docker
+name: default
+
+trigger:
+  ref:
+  - refs/heads/test/**
+  - refs/heads/master
+
+workspace:
+  base: /home/runner/builds
+  path: kubecf
+
+clone:
+  disable: true
+
+# Dockerfile source - https://github.com/f0rmiga/bazel-docker-image.
+.bazel_image: &bazel_image thulioassis/bazel-docker-image:1.2.1
+
+steps:
+- name: clone
+  image: *bazel_image
+  commands:
+  - git clone --depth=2 --branch="${DRONE_BRANCH}" "${DRONE_REMOTE_URL}" .
+  - git checkout "${DRONE_COMMIT}"
+- name: lint
+  image: *bazel_image
+  commands:
+  - ./dev/linters/shellcheck.sh
```

Comment:
Style question:

Would it be better if we had separate steps for these?
```yaml
- name: lint:shellcheck
  image: *bazel_image
  commands: [ dev/linters/shellcheck.sh ]
- name: lint:yaml
  image: *bazel_image
  commands: [ dev/linters/yamllint.sh ]
- name: lint:helm
  image: *bazel_image
  commands: [ dev/linters/helmlint.sh ]
```

Mostly wanting this so that if we need to point somebody at a failure, we can do so showing shorter, more relevant logs.